check-synced.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

Reply:
unless you are positive bash is available. It isn't on bitcoind, for instance. This will be a portability issue if we ever want to get back on alpine.

Reply:
Definitely on bitcoind https://github.com/Start9Labs/bitcoind-wrapper/blob/53b56652c5220d946129ab3104171c5fd62b2451/check-synced.sh#L1

Almost all our packages include `bash` now.

Reply:
k. compat does not...which I think was why this is fresh in my memory.